pkgs/by-name/lu/luaprompt/package.nix

```
@@ -0,0 +1,64 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  nix-update-script,
+  pkg-config,
+  makeWrapper,
+  lua,
+  luaPackages,
+  readline,
+}:
+
+stdenv.mkDerivation rec {
+
+  pname = "luaprompt";
+  version = "0.9";
+
+  src = fetchFromGitHub {
+    owner = "dpapavas";
+    repo = "luaprompt";
+    rev = "v${version}";
+    sha256 = "sha256-S6bzlIY1KlMK3wy01wGuRujGFgPxcNWmCaISQ87EBGs=";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+    makeWrapper
+  ];
+
+  buildInputs = [
+    lua
+    readline
+    luaPackages.argparse
+  ];
+
+  makeFlags = [
+    "PREFIX=${placeholder "out"}"
+    "MANPREFIX=${placeholder "out"}/share/man"
+  ];
+
+  patchPhase = ''
+    substituteInPlace Makefile \
+      --replace-fail "/usr/bin/install" "install" \
+      --replace-fail "VERSION = 5.4" "VERSION = ${lua.luaversion}"
+  '';
+
+  postFixup = ''
+    LUA_CPATH=$out/lib/lua/${lua.luaversion}/?.so;$LUA_CPATH
+    wrapProgram $out/bin/luap \
+      --set LUA_PATH "$LUA_PATH" \
+      --set LUA_CPATH "$LUA_CPATH"
+  '';
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = with lib; {
+    description = "An embeddable Lua command prompt as well as a stand-alone interpreter with pretty-printing and autocompletion";
```

```suggestion
    description = "Embeddable Lua command prompt as well as a stand-alone interpreter with pretty-printing and autocompletion";
```

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes